<launch>
  <!-- 你的硬件与控制器在此先启动（若有） -->
  <!-- <include file="$(find your_hw_pkg)/launch/your_hw.launch"/> -->

  <group ns="right_arm_controller">
    <node pkg="right_arm_hw" type="ftj_to_moveJ_bridge" name="ftj_to_moveJ_bridge" output="screen">
      <param name="stream_rate" value="200"/>
      <!-- 如需改成你控制器实际订阅的话题，在此重映射 -->
      <remap from="command_moveJ" to="/all_joints_hjc/command_moveJ"/>
      <!-- 若要显式指定关节名顺序（不指定则用默认）：
      <param name="joint_names" value="['leg_r1_joint','leg_r2_joint','leg_r3_joint','leg_r4_joint','leg_r5_joint','leg_r6_joint','leg_r7_joint']"/>
      -->
    </node>
  </group>
</launch>
